backend/monitoring/index.js

```
@@ -0,0 +1,151 @@
+  app.use(notFoundHandler());
+  
+  // Central error handler must be last
+  app.use(errorHandler());
+};
+
+module.exports = {
+  initializeMonitoring,
+  setupErrorHandlers,
+  ApiError,
+  asyncHandler,
+  logger,
```

Biome (1.9.4)

[error] This shorthand property named logger is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this shorthand property named logger


(lint/suspicious/noDuplicateObjectKeys)